release/ray_release/byod/requirements_byod_3.8.in

```diff
@@ -12,6 +12,7 @@ gym[atari]
 importlib-metadata
 lightgbm
 memray
+mosaicml-streaming
```

Comment:
this should be added to requirements_byod_ml_3.8.in since your test uses ray-ml image